### PR TITLE
Data.Common tests fixed for non english windows

### PR DESCRIPTION
**Pull request by Anipik**
Fixes https://github.com/dotnet/corefx/issues/27626

### CODE REVIEW

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

File: src/System.Data.Common/tests/System/Data/DataColumnTest2.cs [hunk 1]
```diff
@@ -611,7 +611,7 @@ public void Expression_Simple()
                 if (str == "7.60")
                     str = "7.6";
 
-                Assert.Equal(temp, str);
+                Assert.Equal(double.Parse(temp), double.Parse(str));
```
[the remark this answers is not in the document]
Comment (Anipik) [reply src/System.Data.Common/tests/System/Data/DataColumnTest2.cs]: @stephentoub it fixes almost all the cases(except one). I am parsing the other ones with the IFormatProvider like https://github.com/dotnet/corefx/pull/28872/files#diff-53a5a096471a967cb12442c1d8053298R472

cc @danmosemsft

[the remark this answers is not in the document]
Comment (danmosemsft) [reply src/System.Data.Common/tests/System/Data/DataColumnTest2.cs]: I recommend taking a little longer to find a more robust approach as @stephentoub suggests.

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

File: src/System.Data.Common/tests/System/Data/DataColumnTest2.cs [hunk 2]
```diff
@@ -608,8 +608,8 @@ public void Expression_Simple()
                 else
                     str = dr["expr"].ToString();
 
-                if (str == "7.60")
-                    str = "7.6";
+                if (str == 7.60.ToString())
```
Comment (danmosemsft) [inline src/System.Data.Common/tests/System/Data/DataColumnTest2.cs]: Specify Cultureinfo.CurrentCulture (I assume) explicitly

File: src/System.Data.Common/tests/System/Data/SqlTypes/SqlDecimalTest.cs [hunk 3]
```diff
@@ -253,11 +253,11 @@ public void ArithmeticMethods()
         [Fact]
         public void AdjustScale()
         {
-            Assert.Equal("6464.646400", SqlDecimal.AdjustScale(_test1, 2, false).Value.ToString());
-            Assert.Equal("6464.65", SqlDecimal.AdjustScale(_test1, -2, true).Value.ToString());
-            Assert.Equal("6464.64", SqlDecimal.AdjustScale(_test1, -2, false).Value.ToString());
-            Assert.Equal("10000.000000000000", SqlDecimal.AdjustScale(_test2, 10, false).Value.ToString());
-            Assert.Equal("79228162514264337593543950335.00", SqlDecimal.AdjustScale(_test5, 2, false).ToString());
+            Assert.Equal(6464.646400.ToString(), SqlDecimal.AdjustScale(_test1, 2, false).Value.ToString());
```
Comment (danmosemsft) [inline src/System.Data.Common/tests/System/Data/SqlTypes/SqlDecimalTest.cs]: You should explicitly cast to the type (short or whatever) before calling To String, so you know you will get the same result.

File: src/System.Data.Common/tests/System/Data/DataColumnTest2.cs (hunk 2 above)
Comment (Anipik) [reply src/System.Data.Common/tests/System/Data/DataColumnTest2.cs]: ToString by itself uses CultureInfo.CurrentCulture

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

**Comment by Anipik** [comment]
@dit

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

**Comment by Anipik** [comment]
@dotnet-bot test this please

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

**Comment by Anipik** [comment]
@dotnet-bot test this please

**Comment by Anipik** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build

**Comment by Anipik** [comment]
@dotnet-bot test NETFX x86 Release Build

**Pull request merged**